src/test/java/com/bootexample4/RoostTest/GetAccountInfoPostTest.java

```diff
@@ -0,0 +1,429 @@
+// Test generated by RoostGPT for test test-Koreai using AI Type Vertex AI and AI Model 1957238449580802048
+
+
+// Test generated for /GetAccountInfo_post for http method type POST in rest-assured framework
+
+
+// RoostTestHash=8a95fbc531
+
+
+package com.bootexample4.RoostTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.Test;
+import static io.restassured.RestAssured.given;
+import static org.junit.Assert.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+import org.hamcrest.MatcherAssert;
+import static org.hamcrest.Matchers.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GetAccountInfoPostTest {
+  
+    @Test  
+    public void GetAccountInfo_post_Test() {  
+        RestAssured.baseURI = System.getenv("BASE_URL");  
+  
+        // Read CSV file  
+        try (BufferedReader reader = new BufferedReader(new FileReader("src/test/java/com/bootexample4/RoostTest/GetAccountInfoPostTest.csv"))) {  
+            String headerLine = reader.readLine();  
+            String[] headers = headerLine.split(",");  
+  
+            String line;  
+            while ((line = reader.readLine()) != null) {  
+                String[] data = line.split(",");  
+  
+                // Create a map of header to data  
+                Map<String, String> map = new HashMap<>();  
+                for (int i = 0; i < headers.length; i++) {  
+                    map.put(headers[i], data[i]);  
+                }  
+                
+  
+                Response response = given()
+				.pathParam("koreUserId", map.get("koreUserId"))
+				.pathParam("botId", map.get("botId"))
+				.pathParam("accountId", map.get("accountId"))
+				.pathParam("Authorization", map.get("Authorization"))
+				.pathParam("X-Auth-token", map.get("X-Auth-token"))
+				.pathParam("X-CORRELATION-ID", map.get("X-CORRELATION-ID"))
+				.contentType(ContentType.JSON)
+				.body("{\n"+
+					"  \"customerId\": \"" + map.get("customerId") + "\n" +
+ 				"}")
+                .when()
+                .post("/GetAccountInfo")  
```

Comment:
when passing path paramarers in the url must have placeholders,?
For example it work when we pass
.post("/{koreUserId}/{botId}/{accountId}/{Authorization}/{X-Auth-token}/{X-CORRELATION-ID}")